src/js/components/projects/edit.tsx

```diff
@@ -0,0 +1,108 @@
+import Button from '@salesforce/design-system-react/components/button';
+import Input from '@salesforce/design-system-react/components/input';
+import Modal from '@salesforce/design-system-react/components/modal';
+import Textarea from '@salesforce/design-system-react/components/textarea';
+import i18n from 'i18next';
+import React, { useCallback, useRef } from 'react';
+
+import { Project } from '@/store/projects/reducer';
+
+interface EditModalProps {
+  project: Project;
+  isOpen: boolean;
+  toggleModal: (open: boolean) => void;
+}
+const EditModal = ({ project, isOpen, toggleModal }: EditModalProps) => {
+  const form = useRef(null);
+  const submitButton = useRef<HTMLButtonElement | null>(null);
+
+  const handleSubmit = useCallback((e) => {
+    e.preventDefault();
+    //   const { name, description } = e.target;
+    const { name, description } = form.current;
+    //   value here same as defaultvalue
+    console.log(name, description);
+  }, []);
```

Comment:
@jgerigmeyer I am not clear on how to get the user input on these uncontrolled inputs.  Is this is right approach since we need to set default values from the project? (per: https://github.com/reduxjs/redux/issues/699#issuecomment-139771661) :thinking: 

When I walk through this `handleSubmit` the input values don't change from the defaultValue.

Reply:
None of the existing forms/modals seem to fit our needs for editing existing objects at first glance, but curious if I should attempt to extend one for this purpose?  

My thoughts are the `editModal` should be a utility component (like the `submitModal`) since I believe the interactions for editing other object is similar.